### PR TITLE
Created fetchCrafters function in dataAccess module

### PR DESCRIPTION
**Pull request by seaborndan**
fetchCrafters function made to access data in JSON database and to store it in applicationState

### CODE REVIEW

File: src/scripts/dataAccess.js [hunk 1]
```diff
@@ -59,3 +59,15 @@ export const setIngredients = (id) => {
   // Step 2: If it does, remove it with delete() method
   // Step 3: If it does not, add it with add() method
 };
+
+export const fetchCrafters = () => {
+  return fetch(`${API}/plumbers`)
```
Comment (JoshuaJohns) [inline src/scripts/dataAccess.js:64]: Change plumbers to crafters

**Review by JoshuaJohns — CHANGES_REQUESTED** [review]

File: src/scripts/dataAccess.js (hunk 1 above)
Comment (seaborndan) [reply src/scripts/dataAccess.js]: Yeah got it :)

**Review by JoshuaJohns — APPROVED** [review]
Looks Good

**Review by mvdutta — COMMENTED** [review]
looks great with change

**Review by jacobgt7 — APPROVED** [review]

**Review by mvdutta — APPROVED** [review]

**Pull request merged**